dev-app/src/screens/CollectCardPaymentScreen.tsx

```diff
@@ -186,6 +199,28 @@ export default function CollectCardPaymentScreen() {
       paymentIntent = response.paymentIntent;
       paymentIntentError = response.error;
     } else {
+      const offlineStatus = await getOfflineStatus();
+      let paymentAmountsByCurrency = 0;
+      for (let currency in offlineStatus.sdk.offlinePaymentAmountsByCurrency) {
+        if (currency === inputValues.currency) {
+          paymentAmountsByCurrency =
+            offlineStatus.sdk.offlinePaymentAmountsByCurrency[currency];
+        }
+      }
+      console.log(
+        'currency = ' +
+          inputValues.currency +
+          '  paymentAmountsByCurrency = ' +
+          paymentAmountsByCurrency
+      );
```

Comment:
Is this `console.log` intentionally still in the code here? I don't see any other `console.log` calls in this file, so I suggest removing it.